packages/reports/src/Components/Elements/PositionsDrawerCard/positions-drawer-card.jsx

```diff
@@ -211,19 +215,4 @@ PositionsDrawerCard.propTypes = {
     type: PropTypes.string,
 };
 
-export default connect(({ modules, ui, client, common }) => ({
```

Comment:
we need to pass the props from the parent wrapper since this component will be shared between packages.